cvmfs/swissknife_lease.cc

```diff
@@ -0,0 +1,141 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "swissknife_lease.h"
+
+#include <fstream>
```

Comment:
I can check {string,posix}.cc for which functions to use, but what was the problem is C++ std streams?